src/rgw/rgw_bucket.cc

```diff
@@ -1628,16 +1628,21 @@ int RGWBucketAdminOp::info(RGWRados *store, RGWBucketAdminOpState& op_state,
       return ret;
     }
   } else {
-    RGWAccessHandle handle;
+    void *handle = nullptr;
+    bool truncated = true;
 
     formatter->open_array_section("buckets");
```

Comment:
@cbodley in the future these could be switched to Formatter::ArraySection objects.